.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,79 @@
+name: 🐛 Bug report
+description: Create a report to help us improve
+
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        Before you submit an issue we recommend you visit [Fastify Help](https://github.com/fastify/help)
+        if you have any questions regarding using Fastify.
+
+  - type: checkboxes
+    id: prerequisites
+    attributes:
+      label: Prerequisites
+      options:
+      - label: I have written a descriptive issue title
+        required: true
+      - label: | 
+          I have searched existing issues to ensure it has not already been reported
+        required: true
+
+  - type: input
+    id: fastify-version
+    attributes:
+      label: Fastify version
+      placeholder: 3.x.x
+    validations:
+      required: true
+
+  - type: input
+    id: node-version
+    attributes:
+      label: Node.js version
+      placeholder: 14.x
+    validations:
+      required: true
+
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operating system
+      options:
+        - Linux
+        - macOS
+        - Windows
+    validations:
+      required: true
+
+  - type: input
+    id: os-version
+    attributes:
+      label: Operating system version (i.e. 20.04, 11.3, 10)
+    validations:
+      required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to Reproduce
+      description: |
+        List of steps, sample code or a link to code or a project that reproduces the behavior.
+        Make sure you place a stack trace inside a code (```) block to avoid linking unrelated issues.
```

Comment:
Add a link here to the GitHub formatting documentation. I can say with extreme confidence that there is a vast population of developers who do not understand the triple backticks without some education.

Reply:
Added in https://github.com/fastify/.github/pull/8/commits/0aaac973e9169e901a039f74087eee3d2c6e5350!